src/components/SearchForm/SearchForm.js

```diff
@@ -22,16 +23,17 @@ const SearchForm = () => {
       <form className="text-start" onSubmit={handleSubmit}>
         <label className="form-label" htmlFor="search">Search</label>
         <input name="search" className="form-control" id="search"
+          data-testid="search-input"
           type="text"
           placeholder="Search..."
           value={searchTerm}
-          onChange={handleInputChange}
+          onChange={(e) => { setSearchTerm(e.target.value) }}
```

Comment:
Why do we need onChange and onKeyDown actions?

Reply:
@NikitaUstsiushenka  I've added onKeyDown action to test form submit on "Enter" key press